modules/learn/pages/services-and-indexes/indexes/storage-modes.adoc

```diff
@@ -0,0 +1,63 @@
+= Storage Settings
+:description: All indexes in Couchbase Capella use the Couchbase Plasma storage engine. 
+:page-topic-type: concept
+:page-aliases: indexes:storage-modes,understanding-couchbase:services-and-indexes/indexes/storage-modes,architecture:index-storage
+
+[abstract]
+{description}
+
```

Comment:
Adding this back in because it applies to plasma indexes:

```suggestion

Couchbase Capella indexes implement multi-version concurrency control (MVCC) to provide consistent index scan results and high throughput.
```